### PR TITLE
Add ECMAScript edition selector

### PR DESCRIPTION
**Pull request by jedel1043**
Also adjusted the hide switch to hide the passed tests, since the logic is mostly the same as ES edition filtering.

Some screenshots of the final result:

image

On a tag without versioning (disables the selector):

image

### CODE REVIEW

File: test262/index.html [hunk 1]
```diff
@@ -141,14 +141,28 @@ <h2 class="card-title">Older versions</h2>
       <div class="row">
         <div class="col-md-12" id="progress-info"></div>
       </div>
-      <div class="row">
-        <div class="col-md-12 d-none" id="info-options">
-          <div class="form-check form-switch">
+      <form class="d-none" id="info-options">
+        <div class="form-row d-flex">
+          <div class="form-group">
+            <select class="custom-select" id="info-options-es-version">
```
Comment (Razican) [inline test262/index.html:147]: Why `custom-select` here? I thought Bootstrap had `form-select` to enhance the look of the selection box:

https://getbootstrap.com/docs/5.3/forms/select/

**Review by Razican — APPROVED** [review]
This is a great addition! I just have a comment about the style of the selection box :)

File: test262/index.html (hunk 1 above)
Comment (jedel1043) [reply test262/index.html]: Done!

**Review by HalidOdat — APPROVED** [review]
Nice feature! Looks perfect to me!

**Pull request merged**